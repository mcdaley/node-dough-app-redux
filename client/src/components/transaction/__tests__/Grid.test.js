//-----------------------------------------------------------------------------
// client/src/components/transaction/__tests__/Grid.test.js
//-----------------------------------------------------------------------------
import React            from 'react'
import {
  render,
  cleanup,
}                       from '@testing-library/react'

import TransactionGrid  from '../Grid'

/**
 * TransactionGrid Tests
 */
describe('TransactionGrid', () => {
  const handler       = jest.fn(e => e.preventDefault())
  afterEach(cleanup)
    
  it('Takes a snapshot', () => {
    const { asFragment } = render(
      <TransactionGrid transactions={[]} />
    )
    
    expect(asFragment(
      <TransactionGrid  transactions={[]} />)).toMatchSnapshot()
  })
})