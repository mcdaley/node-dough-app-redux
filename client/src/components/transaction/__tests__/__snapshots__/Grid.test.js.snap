// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TransactionGrid Takes a snapshot 1`] = `
<DocumentFragment>
  <div
    class="react-bootstrap-table"
  >
    <table
      class="table table-striped table-bordered table-condensed"
    >
      <thead>
        <tr>
          <th
            style="width: 15%; text-align: center;"
            tabindex="0"
          >
            Date
          </th>
          <th
            style="width: 25%;"
            tabindex="0"
          >
            Description
          </th>
          <th
            style="width: 15%;"
            tabindex="0"
          >
            Category
          </th>
          <th
            style="width: 15%; text-align: right;"
            tabindex="0"
          >
            Debit
          </th>
          <th
            style="width: 15%; text-align: right;"
            tabindex="0"
          >
            Credit
          </th>
          <th
            style="width: 15%; text-align: right;"
            tabindex="0"
          >
            Balance
          </th>
        </tr>
      </thead>
    </table>
  </div>
</DocumentFragment>
`;
